src/includes/sourcemaps/generate/javascript.svelte.mdx

```diff
@@ -0,0 +1,32 @@
+To generate source maps with your Svelte project, you need to set the sourcemap [compiler options](https://svelte.dev/docs#compile-time-svelte-compile) in your Svelte config:
```

Comment:
If you're using the term "source map" generally here, then go with the edit I've made. If you're trying to reference the sourcemap option in Svelte more directly, then change it to `sourcemap` in code format.

```suggestion
To generate source maps with your Svelte project, you need to set the source map [compiler options](https://svelte.dev/docs#compile-time-svelte-compile) in your Svelte config:
```